docs/vc-spec.md

```diff
@@ -124,27 +166,22 @@ consumes it.  For example, when using [canister signatures](https://internetcomp
 the context contains a time-stamped yet unsigned VC, for which the canister signature will be
 available only at `get_credential`-call.
 
-
-### 3: Get Credential
+### 4: Get Credential
 
 `get_credential`-service issues the actual credential requested by the user.
 
 ```candid
 service : {
-  get_credential : (GetCredentialRequest) -> (GetCredentialResponse) query;
+    get_credential : (GetCredentialRequest) ->
+        (variant { Ok : IssuedCredentialData; Err : IssueCredentialError;}) query;
 };
 
 type GetCredentialRequest = record {
     signed_id_alias : SignedIdAlias;
-    prepared_context : opt blob;
     credential_spec : CredentialSpec;
-};
-type GetCredentialResponse = variant {
-    Ok : IssuedCredentialData;
-    Err : IssueCredentialError;
+    prepared_context : opt vec nat8;
```

Comment:
Why did you change it to `vec nat8`? `blob` is an official candid shorthand for that.

Reply:
Ah, copy-paste from .did-file, sorry!
Reverted the change.